src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,73 @@
+package lotto.domain;
+
+import lotto.exception.IllegalLottoNumberSizeException;
+import lotto.exception.NumberDuplicationException;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicket {
+    public static final int LOTTO_NUMBER_SIZE = 6;
+    public static final int FROM_INDEX = 0;
+    public static final int START_INCLUSIVE = 1;
+    public static final int END_EXCLUSIVE = 46;
```

Comment:
```suggestion
    public static final int END_INCLUSIVE = 45;
```
이렇게 정의 하고 아래쪽와 같이 사용하는 방법도 있습니다
```java
    IntStream.rangeClosed(START_INCLUSIVE, END_INCLUSIVE)...
```

Reply:
변경하고나니 훨씬 더 좋습니다